avoid the check for custom algorithms as it uses git
author: Jarne Renders (Vrije Universiteit Brussel)
diff -ruN flagser.orig/CMakeLists.txt flagser/CMakeLists.txt
--- flagser.orig/CMakeLists.txt	2025-12-01 17:02:24.972483000 +0100
+++ flagser/CMakeLists.txt	2025-12-01 17:03:50.101512000 +0100
@@ -36,13 +36,13 @@
 # Also add the time of the file + output file into consideration
 # Add the include dir to the watch list
 
-add_custom_command(
-    COMMAND ../build_custom_algorithms.sh
-    DEPENDS parser/math_parser.py algorithms.math
-    OUTPUT include/algorithms.h
-    COMMENT "Check for custom algorithms"
-)
-add_custom_target(generate_custom_algorithms DEPENDS include/algorithms.h)
+# add_custom_command(
+#     COMMAND ../build_custom_algorithms.sh
+#     DEPENDS parser/math_parser.py algorithms.math
+#     OUTPUT include/algorithms.h
+#     COMMENT "Check for custom algorithms"
+# )
+# add_custom_target(generate_custom_algorithms DEPENDS include/algorithms.h)
 
 
 add_executable(flagser src/flagser.cpp include/algorithms.h)
